.vue.webpack.config.js

```diff
@@ -0,0 +1,73 @@
+const webpack = require('webpack');
+const path = require('path');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const VueLoaderPlugin = require('vue-loader').VueLoaderPlugin;
+
+const config = {
+    entry: {
+        'vue': './lib/bundles/vue.js',
+    },
+    output: {
+        path: path.resolve(__dirname, 'public/lib/vue'),
+        publicPath: '/public/lib/vue/',
+        asyncChunks: true,
+        clean: true,
+    },
+    module: {
+        rules: [
+            {
+                // Vue SFC
+                test: /\.vue$/,
+                loader: 'vue-loader'
+            },
+            {
+                // Build styles
+                test: /\.css$/,
+                use: [MiniCssExtractPlugin.loader, 'css-loader'],
+            },
+        ]
+    },
+    plugins: [
+        new VueLoaderPlugin(), // Vue SFC support
+        new MiniCssExtractPlugin(),
+        new webpack.ProvidePlugin(
+            {
+                process: 'process/browser', // required by some libs (including `popper.js`)
+            }
+        ),
```

Comment:
Probably not used here.
```suggestion
```

Reply:
After building without it there was an error related to this in the console when loading any component.